openpype/pipeline/create/context.py

```diff
@@ -1458,6 +1458,65 @@ def create(self, identifier, *args, **kwargs):
                     identifier, label, exc_info, add_traceback
                 )
             ])
+        return result
+
+    def create(
+        self,
+        creator_identifier,
+        variant=None,
+        asset_doc=None,
+        task_name=None,
+        pre_create_data=None
+    ):
+        """Trigger create of plugins with standartized
+
+        Args:
+            creator_identifier (str):
+            variant (str): Variant used for subset name.
+            asset_doc (Dict[str, Any]):
+            task_name (str): Name of task to which is context related.
+            pre_create_data (Dict[str, Any]): Pre-create attribute values.
+
+        Returns:
+            Any: Output of triggered creator's 'create' method.
+
+        Raises:
+            CreatorsCreateFailed: When creation fails.
+        """
+
+        if pre_create_data is None:
+            pre_create_data = {}
+
+        project_name = self.project_name
+        if asset_doc is None:
+            asset_name = self.get_current_asset_name()
+            asset_doc = get_asset_by_name(project_name, asset_name)
+            task_name = self.get_current_task_name()
+
+        creator = self.creators.get(creator_identifier)
+        family = None
+        subset_name = None
+        if creator is not None:
+            family = creator.family
+            subset_name = creator.get_subset_name(
+                variant,
+                task_name,
+                asset_doc,
+                project_name,
+                self.host_name
+            )
+        instance_data = {
+            "asset": asset_doc["name"],
+            "task": task_name,
+            "variant": variant,
+            "family": family
```

Comment:
I think this is wrong - we shouldn't need to pass the family explicitly. The creator already produces that data, no? If not - that already seems invalid from the Creator's perspective, no?

Reply:
Based on this PR https://github.com/ynput/OpenPype/pull/4392 the instance does not need to have creator avalable.

Reply:
> Based on this PR https://github.com/ynput/OpenPype/pull/4392 the instance does not need to have creator avalable.

Please explain. :) A Creator doesn't need to specify a family?

Reply:
> Please explain. :) A Creator doesn't need to specify a family?

No, `CreatedInstance` does not require to have passed `CreatorPlugin` in arguments.

Reply:
Oh lord, the family is required argument for `CreatedInstance`. Will change.